lightning/src/onion_message/messenger.rs

```diff
@@ -1178,7 +1183,7 @@ where
 			.collect::<Vec<_>>();
 
 		self.message_router
-			.create_blinded_paths(recipient, peers, secp_ctx)
+			.create_blinded_paths(recipient, None, peers, secp_ctx)
```

Comment:
Updated the ResponseInstruction::WithReplyPath() and `messenger.rs`' create_blinded_path to take MessageContext as input field in [9d7a3bf](https://github.com/lightningdevkit/rust-lightning/commit/9d7a3bf8f86fbb64354ffe5511719900a134c668)
This will allow us to pass the context here!
Thanks for the suggestion!